README.md

```diff
@@ -3,6 +3,12 @@
 
 A crawler to find websites that exercise code in Firefox that is not covered by unit tests
 
+## Software requirements
+
+- You will need python>=3.6, pip for python>=3.6 and [Mercurial](https://www.mercurial-scm.org/) source control management tools installed on your machine to run the script.
```

Comment:
Just list the requirements, no need for the longer description.
```
- Python 3.6+
- [Mercurial](https://www.mercurial-scm.org/)
```

Not needed to specify `pip` as it's basically implied.

Reply:
Yep